BoxBrowseSDK/BoxBrowseSDK/Helpers/BOXBrowseSDKFileTypeHelper.m

```diff
@@ -15,7 +15,7 @@ @implementation BOXBrowseSDKFileTypeHelper
     static NSSet *extensions = nil;
     
     if (extensions == nil) {
-        extensions = [NSSet setWithObjects:@"aac", @"aiff", @"m3u", @"m4a", @"mid", @"mp3", @"wav", @"wpl", @"wma", @"amr", @"3gp", nil];
+        extensions = [NSSet setWithObjects:@"aac", @"aiff", @"m3u", @"m4a", @"mid", @"mp3", @"wav", @"wpl", @"wma", @"amr", @"3gp", @"alac", @"flac", @"ogg", @"m4r", nil];
```

Comment:
maybe exclude ogg from this for now

Reply:
public static function video_formats()
    {
        return ['3g2', '3gp', 'avi', 'm2v', 'm2ts', 'm4v', 'mkv', 'mov', 'mp4', 'mpeg', 'mpg', 'ogg', 'mts', 'qt', 'wmv'];
    }
    /**
     * @return array
     */
    public static function mobile_video_formats()
    {
        return ['flv'];
    }
    /**
     * @return array
     */
    public static function audio_formats()
    {
        return ['aac', 'aif', 'aifc', 'aiff', 'amr', 'au', 'flac', 'm4a', 'mp3', 'ra', 'wav', 'wma'];
    }

Reply:
^^ From Representations.php of the web app